data/tangorishi.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "Rishi Joshi",
+  "type": "personal",
+  "bio": "Just a guy passionate for Web Development",
+
+  "tags": [ "C++","Open Source", "WebDev" "Javascript", "Java", "C", "Frontend"],
```

Comment:
i think accidentally you miss a comma in line (6) between "WebDev" and "Javascript" tag. `"tags": [ "C++","Open Source", "WebDev", "Javascript", "Java", "C", "Frontend"],`

Reply:
yes corrected it.